Classes/Domain/Repository/Localization/LocalizationRepository.php

```diff
@@ -71,4 +74,29 @@ protected static function getBackendUserAuthentication()
     {
         return $GLOBALS['BE_USER'] ?? null;
     }
+
+    public static function getLanguageOverlayRecord($table, $uid, $language = null)
+    {
+        $hiddenField = $GLOBALS['TCA'][$table]['ctrl']['enablecolumns']['disabled'] ?? 'hidden';
+        if ($language === null) {
+            $language = self::getCurrentLanguage();
+        }
+        $baseRecord = BackendUtility::getRecordWSOL($table, $uid, '*', ' AND ' . $hiddenField . '=0');
```

Comment:
IMHO this is a bit wrong.
The extra SQL part will be used by getting the base version which later on will be overloaded with the Workspace or Language version of the record. (Is this still like that?)
So this leads to wrong results, if it is visible (not hidden) in Workspace/Language but on base version.

Reply:
If a L=0 record is hidden their translations L=1 shouldn’t be shown (in default language overlay/fallback mode).
This is core and extbase behavior, thus should be the same here. I stumbled upon this case in regression tests while moving from css_styled_content+Typoscript to Fluid+TVP rendering for some elements. 
The TVP+Fluid solution suddenly showed a lot of such rogue elements (translations non-hidden, base language hidden) which were not output before.

Reply:
We should have this in mind

Reply:
I do, that's why I planned https://github.com/T3Voila/templavoilaplus/pull/432 as a general interface allowing to manage every language aspect in a centralised manner. E.g. to add all assumptions there in order to make them configurable later ;-)